.github/workflows/deploy.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy `dandidav` to Heroku
+
+on:
+  release:
+    types: [released]
+
+concurrency:
+  # If this workflow is already running, cancel it to avoid a scenario
+  # where the older run finishes *after* the newer run and overwrites
+  # the deployment with an older version of the app.
+  group: ${{ github.workflow }}
+  cancel-in-progress: true
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  deploy:
+    name: Deploy to Heroku
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # fetch history for all branches and tags
+
+      - name: Compile the Rust appication
+        run: cargo build --release
+
+      - name: Install Heroku CLI
+        run: curl https://cli-assets.heroku.com/install.sh | sh
+
+      - name: Install builds plugin
+        run: heroku plugins:install heroku-builds
+
+      - name: Create Heroku Build
+        run: heroku builds:create --app dandidav
```

Comment:
Is this deploying the entire repository, including the whole of the `target/` directory?  Can we not just deploy only the `target/release/dandidav` asset?

Reply:
Unfortunately the `heroku builds` plugin doesn't allow specifying a single asset. And additionally, you have to include the `Procfile`. 

I didn't realize how many files got stored in the `target/` directory until I just ran the compilation locally, and I agree it's not ideal to push all those artifacts to Heroku. I tried getting around it by storing things in a temporary directory and pushing that - https://github.com/dandi/dandidav/pull/105/commits/6b8df2c06453999b1a2bca17796afb1d1452f0b1